.github/workflows/examples_js.yml

```diff
@@ -0,0 +1,92 @@
+name: JS Example Tests
+
+on:
+  workflow_call:
+    inputs:
+      working-directory:
+        required: true
+        type: string
+      api_key:
+        required: false
+        type: string
+      base_url:
+        required: false
+        type: string
+      author:
+        required: false
+        type: string
+      commit_message:
+        required: false
+        type: string
+      dont_notify:
+        required: false
+        type: boolean
+  workflow_dispatch:
+    inputs:
+      working-directory:
+        required: true
+        type: string
+      api_key:
+        required: false
+        type: string
+      base_url:
+        required: false
+        type: string
+      author:
+        required: false
+        type: string
+      commit_message:
+        required: false
+        type: string
+      dont_notify:
+        required: false
+        type: boolean
+
+jobs:
+  run-js-tests:
+    name: Run JS Example Tests
+    defaults:
+      run:
+        working-directory: ${{ inputs.working-directory }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        node-version: ["18"]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
```

Comment:
`actions/setup-node@v4`  There might be caching or other issue in older version.